internal/operator/internal/index_image.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+
+	"github.com/operator-framework/operator-sdk/internal/operator"
+)
+
+type IndexImageCatalogCreator struct {
+	IndexImage       string
+	InjectBundles    []string
+	InjectBundleMode string
+
+	cfg *operator.Configuration
```

Comment:
Wondering if we could add DBPath to either `Configuration` struct or `IndexImageCatalogCreator`. We need that (other than BundleImage and Namespace) to init and create the registry pod.

Reply:
I think getting the db path is an internal implementation detail of the `IndexImageCatalogCreator`. We can read the image labels on `IndexImage` to find the DB path. I can update this PR to handle that piece.